src/test/java/org/cactoos/text/ComparableTextTest.java

```diff
@@ -68,4 +71,64 @@ public void comparesToUncheckedText() {
             Matchers.equalTo(0)
         );
     }
+
+    @Test
+    public void equalsToItself() {
+        final Text text = new TextOf("text");
+        MatcherAssert.assertThat(
+            "Does not equal to itself",
+            text,
+            new IsEqual<>(
```

Comment:
@ilyakharlamov Use `TextIs` instead `IsEquals`

Reply:
@fabriciofx TestIs matcher calls ComparableText.asString(), while we are talking about the ComparableText.equals() method here, this will lead the test to check the wrong thing and produce false-positive results. I've added both IsEqual and TextIs

Reply:
@ilyakharlamov Right. I agree.